db/payment_icons.yml

```diff
@@ -242,3 +242,19 @@
   name: dailyyamazaki
   label: Daily Yamazaki
   group: convenience_stores
+-
+  name: mondido
+  label: Mondido
+  group: other
+-
+  name: collector_bank
+  label: Collector
```

Comment:
I don't have much context on this, but should the label by any chance be "Collector Bank"?